package.json

```diff
@@ -4,19 +4,20 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "node index.js",
+    "start-watch": "nodemon --inspect=0.0.0.0 --ext js --legacy-watch index.js",
     "lint": "standard",
-    "unit-test": "lab -c -r console -o stdout -r lcov -o lcov.info",
-    "test": "npm run lint && npm run unit-test",
+    "pretest": "npm run lint",
+    "test": "lab -l -c -r console -o stdout -r junit -o ./test-output/junit.xml",

```

Comment:
There is a memory leak in Sequelize from one of the referenced libraries 'any-promise'.
This causes the tests to fail unless the `-l` switch is used to disable failing due to memory leaks.

Sequelize doesn't use native promises, but relies on bluebird promises. Apparently the leak is related to 'longStackTraces', but the leak remained when attempting to disabling them, or switch to native promises.

It's cause for concern and needs futher investigation